src/bitmap.c

```diff
@@ -1885,7 +1886,7 @@ GdipBitmapLockBits (GpBitmap *bitmap, GDIPCONST GpRect *rect, UINT flags, PixelF
 		return InvalidParameter;
 	}
 
-	if (!gdip_is_a_supported_pixelformat (format))
+	if (format != src_data->pixel_format && !gdip_is_a_supported_pixelformat (format))
```

Comment:
`test_createBitmapFromScan016bppRGB555:25`: `GdipBitmapLockBits` failed. Fixed by adding `format != src_data->pixel_format` check in `GdipBitmapLockBits` argument validation